roles/openshift_logging/defaults/kibana.yml

```diff
@@ -0,0 +1,3 @@
+---
+kibana_hostname: kibana.example.com
+kibana_ops_hostname: kibana-ops.example.com
```

Comment:
So what is `openshift_master_logging_public_url` currently being set to for logging installations? Is it different than `openshift_hosted_logging_master_public_url`?